keyboards/handwired/scottokeebs/scottomacrodeck/info.json

```
@@ -0,0 +1,50 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "ScottoMacrodeck",
+    "maintainer": "joe-scotto",
+    "bootloader": "caterina",
```

```suggestion
    "development_board": "promicro",
```
This specifies that it's an atmega32u4, and uses the caterina bootloader (as well as being pro micro pin compatible, for converters)